script.js

```diff
@@ -0,0 +1,100 @@
+//Variables
+let todos = []
+
+//Element storage
+$addBtn = $('#new-task')
+$newTaskInput = $('#new-task-input')
+$addTask = $('#add-task')
+$taskList = $('#todo-list')
+$linkList = $('#link-list')
+
+//Click listeners
+$addBtn.on('click', showForm)
+$addTask.on('click', addTask)
+
+
+
+//on page load, get list of todos from localstorage
+$(document).ready(function(){
+    todos = JSON.parse(localStorage.getItem('todos'))
+    if (!todos) todos = [];
+    renderList()
```

Comment:
if() should have a bracket after I believe? like if (){}

Reply:
I'm pretty sure an if statement doesn't need {} if there is only 1 line following. it will implicitly run that one line.